addons/mail/static/src/core/common/message_actions.js

```diff
@@ -73,6 +73,13 @@ messageActionsRegistry
         title: _t("Delete"),
         onClick: (component) => component.onClickDelete(),
         sequence: 90,
+    })
+    .add("download", {
+        condition: (component) => component.message.attachments.length > 1,
+        icon: "fa-download",
+        title: _t("Download All"),
+        onClick: (component) => component.onClickDownloadAll(),
+        sequence: 100,
```

Comment:
- `download` => `download_files`
- "Download All" => "Download Files"
- `onClickDownloadAll` => `onClickDownloadFiles`

It should be clearer the message action is related to files/attachments, not the message itself as a whole